<ns0:entry xmlns:ns0="http://www.w3.org/2005/Atom" xmlns:ns1="http://schemas.google.com/g/2005" xmlns:ns2="http://schemas.google.com/projecthosting/issues/2009" ns1:etag="W/&quot;CUAARH47eCl7ImA9WhZTEUs.&quot;">
		<ns0:id>http://code.google.com/feeds/issues/p/nativeclient/issues/876/comments/full/18</ns0:id><ns0:author>
			<ns0:name>bugdro...@chromium.org</ns0:name><ns0:uri>/u/101788990881243245974/</ns0:uri></ns0:author><ns0:content type="html">The following revision refers to this bug:
    http://src.chromium.org/viewvc/chrome?view=rev&amp;revision=4528

------------------------------------------------------------------------
r4528 | mseaborn@chromium.org | Mon Mar 14 21:31:47 PDT 2011

Changed paths:
 M http://src.chromium.org/viewvc/chrome/trunk/src/native_client/tools/REVISIONS?r1=4528&amp;r2=4527&amp;pathrev=4528

Update nacl-glibc revision

This pulls in:
fa7664f  Fix some validation failures in x86-32 libm.so

BUG=http://code.google.com/p/nativeclient/issues/detail?id=876
TEST=python build.py -b

Review URL: http://codereview.chromium.org/6694015
------------------------------------------------------------------------</ns0:content><ns0:updated>2011-03-15T04:35:45.000Z</ns0:updated><ns0:published>2011-03-15T04:35:45.000Z</ns0:published><ns2:updates /><ns0:title>Comment 18 by bugdro...@chromium.org</ns0:title><ns0:link href="http://code.google.com/p/nativeclient/issues/detail?id=876#c18" rel="alternate" type="text/html" /><ns0:link href="https://code.google.com/feeds/issues/p/nativeclient/issues/876/comments/full/18" rel="self" type="application/atom+xml" /></ns0:entry>